ScreenToGif/Windows/Other/DownloadDialog.xaml.cs

```diff
@@ -171,10 +171,6 @@ private void Window_Loaded(object sender, RoutedEventArgs e)
 
             #endregion
 
-            //If cancelled.
-            if (!IsLoaded)
```

Comment:
This one isn't redundant. If the window gets closed between line 125 and 173 (unzip files), there's another chance to stop the task. I'm going to accept the PR anyway.